mmv1/third_party/terraform/services/databasemigrationservice/resource_database_migration_service_connection_profile_test.go

```diff
@@ -40,6 +45,39 @@ func TestAccDatabaseMigrationServiceConnectionProfile_update(t *testing.T) {
 	})
 }
 
+func TestAccDatabaseMigrationServiceConnectionProfile_Postgres_PSC(t *testing.T) {
+	t.Parallel()
+
+	instanceName := "tf-test-" + acctest.RandString(t, 10)
+	projectId := "psctestproject" + acctest.RandString(t, 10)
+	orgId := envvar.GetTestOrgFromEnv(t)
+	billingAccount := envvar.GetTestBillingAccountFromEnv(t)
+	certName := "sqlcert" + acctest.RandString(t, 10)
+	userName := "username" + acctest.RandString(t, 10)
+	passWord := "password" + acctest.RandString(t, 10)
+	profileName := "dbmsprofile" + acctest.RandString(t, 10)
+	profileDisplay := "profiledisplay" + acctest.RandString(t, 10)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccSqlDatabaseInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatabaseMigrationServiceConnectionProfile_Postgres_PSC(instanceName, projectId, orgId, billingAccount, certName, userName, passWord, profileName, profileDisplay),
+				Check:  resource.ComposeTestCheckFunc(verifyPscOperation("google_sql_database_instance.instance", true, true, []string{envvar.GetTestProjectFromEnv()})),
```

Comment:
Why does this need to happen as a check func? Why is the ImportStateVerify check insufficient?

Reply:
I took that part of the test from the SQL Instance test where I got the format to create a PSC enabled instance: https://github.com/GoogleCloudPlatform/magic-modules/blob/6bc25c325371b1bd53bb041269688a285274a506/mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go#L790

I did not question if that would be needed here -- I assumed if it was needed for there it probably would be for a different test that created one, but since it isn't the primary resource being tested here I could try removing it.

Reply:
work moved to https://github.com/GoogleCloudPlatform/magic-modules/pull/9137